src/di.php

```diff
@@ -493,6 +493,15 @@
     }
 };
 
+$di['set_redirect_uri'] = function () use ($di) {
```

Comment:
It's pretty nitpicky, I know, but I think that `set_return_uri` better fits the actual usecase of this function